.github/workflows/release-docker.yml

```diff
@@ -103,7 +67,7 @@ jobs:
           push: true
           platforms: |
             linux/amd64
-            linux/arm64
+          # linux/arm64 TODO: enable after we can limit parallelization to 1
```

Comment:
enable docker multi-platform build

Currently disabled due to parallelization causing out-of-memory (exit code 137).

Reply:
Open new issue https://github.com/DeFiCh/metachain/issues/165